analyses/cell-type-wilms-tumor-06/notebook_template/03_clustering_exploration.Rmd

```diff
@@ -0,0 +1,722 @@
+
+```
+
+### Compare annotations between them
+
+
+#### Annotation versus `Seurat` clusters
+
+Here we check wether one (or more) `Seurat` cluster do correspond to one label. 
+This would allow us to be more confident in the annotation of immune or endothelial cells for example.
+
+
+```{r fig.width=20, fig.height=15, out.width='100%'}
+p1 <- do_Table_Heatmap( srat,
+                            last_group = "fetal_kidney_predicted.compartment",
+                            first_group = "seurat_clusters"
+                             
+                            )
+
+
+p2 <- do_Table_Heatmap( srat,
+                            last_group = "fetal_full_predicted.annotation.l1",
+                            first_group = "seurat_clusters"
+                            )
+
+
+p3 <- do_Table_Heatmap( srat,
+                            last_group = "singler_celltype_annotation",
+                            first_group = "seurat_clusters"
+                            )
+
+
+p4 <- do_Table_Heatmap( srat,
+                            last_group = "cellassign_celltype_annotation",
+                            first_group = "seurat_clusters"
+                            )
+
+(p1|p2)/(p3|p4)
+```
+
+#### Comparison of the labels from the two fetal references
+
+The use of the right reference is crucial for label transfer and annotation. 
+It is recommended that the cell types in the reference is representative to the cell types to be annotated in the query.
+
+Wilms tumors can contain up to three histologies that resemble fetal kidney: blastema, stroma, and epithelia [[1](https://www.ncbi.nlm.nih.gov/books/NBK373356/), [2](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9915828/)]. 
+Because of their histological similarity to fetal kidneys, Wilms tumors are thought to arise from developmental derangements in embryonic renal progenitors.
+
+In the previous step of the analysis, we selected two fetal references containing kidney samples and transfer the labels. 
+
+Here we aim to understand the overlap and differences between the two references. 
+
+##### Human fetal kidney atlas Stewart et al.
+
+Stewart et al. created a [human fetal kidney atlas](https://www.kidneycellatlas.org/) [[3](https://www.science.org/doi/10.1126/science.aat5031)]. 
+
+This reference contains only fetal kidney cells and has been precisely annotated by kidney experts. 
+
+##### Azimuth Human fetal atlas Cao et al.
+
+`Azimuth` also provides a human fetal atlas as a reference [[4](https://www.science.org/doi/10.1126/science.aba7721)].
+
+The data can be found on [Zenodo](https://zenodo.org/records/4738021#.YJIW4C2ZNQI).
+
+The reference contain cells from 15 organs including kidney from fetal samples.
+
+We were first nicely surprised that most of the cells in the Wilms tumor dataset mapped to "kidney" organ (see previous report `02a_fetal_all_reference_Cao_{sample_id}.html`). 
+
+##### first level annotation
+
+```{r fig.width=20, fig.height=8, out.width='100%'}
```

Comment:
these are a bit small, so let's increase the height
```suggestion
```{r fig.width=20, fig.height=15, out.width='100%'}
```